src/directConnect.ts

```diff
@@ -43,26 +43,30 @@ export function openDirectConnectionForm(): void {
     // logger.debug("creating connection from form data:", body);
 
     let kafkaConfig: KafkaClusterConfig | undefined = undefined;
-    if (body["bootstrap_servers"]) {
-      kafkaConfig = {
-        bootstrap_servers: body["bootstrap_servers"],
-      };
+    if (body["clusterConfig"]) {
+      kafkaConfig = { ...body["clusterConfig"] };
     }
 
     let schemaRegistryConfig: SchemaRegistryConfig | undefined = undefined;
-    if (body["uri"]) {
-      schemaRegistryConfig = {
-        uri: body["uri"],
-      };
+    if (body["schemaConfig"]) {
+      schemaRegistryConfig = { ...body["schemaConfig"] };
     }
 
     const manager = DirectConnectionManager.getInstance();
-    return await manager.createConnection(
+    const result = await manager.createConnection(
       kafkaConfig,
       schemaRegistryConfig,
       body["name"],
       body["platform"],
     );
+    if (result.success) {
+      await window.showInformationMessage(`🎉 New Connection Created`, {
+        modal: true,
+        detail: `View and interact with ${body["name"]} in the Resources sidebar`,
+      });
+      directConnectForm.dispose();
+    }
```

Comment:
New: close the form and show a modal if we successfully added the connection

Reply:
Just personal preference speaking, but do we need this to be a modal? Not much action the user can / needs to take as a result, and we're already refreshing the Resources view on successful creation. 🤔

Reply:
Open to changing it! Let's review in the DTX meeting. I tried the non-modal info message but felt it was too subtle, and with the form closing it's almost jarring -- like you don't know what happened. I think it was for a similar reason that we switched the Scaffold form success msg to a modal as well.

Reply:
Makes sense - in the project scaffolding scenario it seems like it's easier to lose generated files if they aren't opened in the current / a new window, but in this case the Resources view is likely still open/visible.

Another idea (for a follow-on branch) is to add a button to the notification  to specifically show/highlight it (like we do for freshly-uploaded schemas), but would need some extra logic for the "reveal" functionality in the view provider.